plugins/dotplot-view/src/DotplotView/components/ImportForm.tsx

```diff
@@ -135,6 +135,7 @@ export default () => {
               name="URL"
               description=""
               location={trackData}
+              // @ts-ignore
               setLocation={setTrackData}
```

Comment:
any way we can avoid this ts-ignore?

Reply:
I could revert this change https://github.com/GMOD/jbrowse-components/pull/1505/files#diff-258d5d78c953b6718af9fceea4976ae26a72c21297f3a1ab59245145c7a6c5f7R37 however it is more accurate to type the function instead of what it was

Reply:
To properly solve it would maybe mean re-assessing some details of the FileLocation typescript base